frontend/src/components/student/AllActivities/SurveyTask/SurveyTask.js

```diff
@@ -43,98 +63,117 @@ function FeedbackTask(props) {
-                            ) : (
-                              <span>Wyślij</span>
-                            )}
-                          </FormButton>
-                        </Col>
-                      </Row>
-                    </Container>
-                  </Form>
-                )}
-              </Formik>
-            </HeaderCol>
-          </ActivityCol>
-        )}
-      </InfoContainer>
-    </Content>
+  return (
+    <>
+      <Row className={'m-0 pt-2'}>
+        <Col>
+          <GameCard
+            headerText={'Informacje o aktywności'}
+            content={
+              task === undefined ? (
+                <Loader />
+              ) : task == null ? (
+                <p>{ERROR_OCCURRED}</p>
+              ) : (
+                <CustomTable
+                  $borderColor={props.theme.secondary}
+                  $background={props.theme.secondary}
+                  $fontColor={props.theme.font}
+                  style={{ border: 'none' }}
+                >
+                  <tbody>
+                    {tableContent.map((row, index) => (
+                      <tr key={index}>
+                        <th width={'25%'}>{row.header}</th>
+                        <td>{row.body}</td>
+                      </tr>
+                    ))}
+                  </tbody>
+                </CustomTable>
+              )
+            }
+          />
+        </Col>
+      </Row>
+      <Row className={'m-0 pt-2'}>
+        <Col>
+          <GameCard
+            headerText={'Twoja odpowiedź'}
+            content={
+              task === undefined ? (
+                <Loader />
+              ) : task == null ? (
+                <p>{ERROR_OCCURRED}</p>
+              ) : (
+                <Form>
+                  <FormGroup>
+                    <FormLabel>Twoja opinia (wymagane)</FormLabel>
+                    <FormControl
+                      as={'textarea'}
+                      ref={feedbackRef}
+                      rows={3}
+                      defaultValue={task.feedback.feedback}
+                      style={{
+                        color: props.theme.font,
+                        backgroundColor: props.theme.secondary,
+                        borderColor: props.theme.primary
+                      }}
+                    />
+                  </FormGroup>
+                  <FormGroup className={'my-3'}>
+                    <FormLabel>Twoja ocena (wymagane)</FormLabel>
+                    <div className={'d-flex justify-content-between w-100 h5 mt-3'}>
+                      {icons.map((icon, index) => (
+                        <FontAwesomeIcon key={index} icon={icon} />
+                      ))}
+                    </div>
+                    <RangeSlider
+                      ref={rateRef}
+                      type={'range'}
+                      min={1}
+                      max={5}
+                      defaultValue={task.feedback.rate}
```

Comment:
W każdej z tych wartości musimy sprawdzić najpierw czy `feedback != null` - to jest domyślny stan jeśli nic nie jest oddane, i bez tego aplikacja się wysypie:
![firefox_hmXjKJ7e6F](https://user-images.githubusercontent.com/56168780/196970654-81e778dc-03fe-4fd7-8c68-fcc5417e84b6.png)